src/components/behandlerdialog/BehandlerdialogContainer.tsx

```diff
@@ -0,0 +1,24 @@
+import React, { ReactElement } from "react";
+import Side from "@/sider/Side";
+import { Menypunkter } from "@/navigation/menypunkterTypes";
+import Sidetopp from "@/components/Sidetopp";
+import SideLaster from "@/components/SideLaster";
+import { BehandlerdialogSide } from "@/components/behandlerdialog/BehandlerdialogSide";
+
+const texts = {
+  title: "Dialog med behandler",
+};
+
+export const BehandlerdialogContainer = (): ReactElement => {
```

Comment:
Nå blir det jo likt som de andre sidene våre, men det er egentlig ikke noe poeng i å ha et "Container"-lag ytterst lenger 🤔

Reply:
Enig at vi egentlig ikke trenger det slik som før (for å hente data og sende videre), men synes det er kjekt å ha wrappingen i de <Side*>komponentene i en egen "container"-komponent og så selve side-innholdet i en annen. Gjør det også lettere å skrive test for selve siden isolert hvis man ønsker det.

Reply:
Godt poeng! Jeg tror det hadde vært fint med andre navn, så vi slipper side i side. Men det kan man jo gjøre som en egen greie for alle `Container`ene våre 👍